Toric/ConvexAnalysis/PolyhedralCone.lean

```diff
@@ -19,11 +19,27 @@ namespace PointedCone
 section OrderedSemiring
 variable [Semiring R] [PartialOrder R] [IsOrderedRing R] [AddCommMonoid E] [Module R E] {s : Set E}
 
+theorem span_le (c : PointedCone R E) {s : Set E} :
```

Comment:
Do we need this alias? I think there is no need to copy `Submodule.foo` lemmas into the `PointedCone` namespace.